### PR TITLE
fix: add some companies and their stock symbols

### PR DESCRIPTION
**Pull request by Jason3S**

### CODE REVIEW

File: packages/companies/companies.txt [hunk 1]
```diff
@@ -1,86 +1,166 @@
 2GIS
+3M
 3M Company
+A.O. Smith Corp
 A.P. Moller-Maersk Group
+AAPL
 ABB
 Abbott Laboratories
+Abbott Laboratories
```
Comment (nschonni) [inline packages/companies/companies.txt:9]: Looks like there might be a bunch of dupes from this

Comment (Jason3S) [reply packages/companies/companies.txt:9]: Thanks, I missed that.

Comment (nschonni) [reply packages/companies/companies.txt:9]: Might make sense to strip punctuation if you're doing this through a script

**Comment by nschonni** [comment]
Maybe also strip all the "Corporation", "Inc", "LTD", etc... from the names, since there are probably another chunk that are duplicate variations on those

File: packages/companies/companies.txt (hunk 1 above)
Comment (Jason3S) [reply packages/companies/companies.txt]: A few duplicates won't hurt. They are removed during the preprocessing.

**Comment by Jason3S** [comment]
> Maybe also strip all the "Corporation"/"Corp", "Inc"/"Incorporated", "LTD"/"Limited", etc... from the names, since there are probably another chunk that are duplicate variations on those

I'm not as worried about those even though they are the same thing.

**Pull request merged**